lib/comply/whitelist_constantize.rb

```diff
@@ -0,0 +1,30 @@
+require 'active_model/validations'
+
+module Comply
+  module WhitelistConstantize
+    class NotWhitelistedError < ArgumentError
+    end
+
+    DEFAULT_PROC = -> (klass) { klass <= ActiveModel::Validations }
+
+    class << self
+      def constantize(str)
+        whitelist! { str.classify.constantize }
+      end
+
+      def whitelist
```

Comment:
Do you add an additional whitelist element by doing `WhitelistConstantize.whitelist << -> { ... }`? Might be nice to have a method like `add_to_whitelist` or something to encapsulate this behavior.